calling-apis/intro-to-asynchronous-programming.md

```diff
@@ -0,0 +1,136 @@
+# Intro to Asynchronous Programming
+
+## Learning Goals
+
+- Define asynchronous programming
+- Compare synchronous and asynchronous programming
+
+## Introduction: A Pizza Metaphor
+
+It's Friday, and Mica is exhausted and spending the evening at home. It's dinner time, and they have no food in their freezer nor fridge. They aren't going to go grocery shopping, and nobody will make dinner for them tonight.
+
+Instead, they'll make a phone call to their favorite restaurant (who's not on an app) and order dinner for themself for delivery.
+
+There's a series of things that happen between the moment Mica makes the order all the way to the point where they take the first bite.
+
+In pain-staking detail, let's outline all the actions Mica would do, beginning with picking up the phone, and ending with eating the meal.
+
+<br/>
+
+<details style="max-width: 700px; margin: auto;">
+  <summary>
+    No, seriously. Take one minute to make a list of the actions Mica would do. After that, then read more here.
+  </summary>
+
+We may imagine that what Mica does is make the order, then wait for the order, and then eat.
+
+It's actually way more likely that Mica will make the order, then **before the order arrives, they do other things,** like read, shower, sleep, watch TV. Mica is able to eat their food only after the food order has been received, cooked, assembled, assigned a driver, driven over, and delivered.
+
+The food order was received, made, and delivered. However, while that was happening, Mica **did not wait for the food order to finish** in order to continue other actions. While waiting for the food, Mica didn't stop everything they were doing, sit on the couch, and stare at the wall. (Or maybe they did!)
+
+<br/>
```

Comment:
Feel free to ignore, I just don't like unneeded HTML.

```suggestion
```

Reply:
+1

This one results in a strangely tall gap between these two paragraphs.